src/rqt_console/message_data_model.py

```diff
@@ -40,7 +40,7 @@
 class MessageDataModel(QAbstractTableModel):
 
     # the column names must match the message attributes
-    columns = ['message', 'severity', 'node', 'stamp', 'topics', 'location']
```

Comment:
Unlike ROS1, It seems ROS2 log messages don't have `topics` entity, so I dropped it.